docs/reference/setup/install/connect-clients.asciidoc

```diff
@@ -1,13 +1,16 @@
 ==== Connect clients to {es}
 
-When you start {es} for the first time, TLS is configured automatically for the 
-HTTP layer. A CA certificate is generated and stored on disk, and the hex-coded 
-SHA-256 fingerprint of this certificate is also output to the terminal. Any 
-clients that connect to {es}, such as the 
+When you start {es} for the first time, TLS is configured automatically for the
```

Comment:
@jkakavas Can you take another look? I thought it made sense to mention the file path where we talk about the file being stored on disk. 

Also...is hex-coded correct? Or should this be hex-encoded?

Reply:
Good catch ! It’s hex-encoded